src/data/thermal-conductivity-constants.ts

```diff
@@ -90,7 +90,7 @@ export const DIPPRThermalConductivityDictionary: Record<
     C1: 0.33363,
     C2: -0.00083655,
     minimumTemperature: 192.4,
-    maximumTemperature: 250,
+    maximumTemperature: 2,
```

Comment:
Why only first digit?

Reply:
ooo, I made some scipting mistake , i will correct it